sources/models/src/modeled_types/kubernetes.rs

```diff
@@ -1002,6 +1006,177 @@ mod test_topology_manager_policy {
 
 // =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
 
+/// imageGCHighThresholdPercent is the percent of disk usage after which image
+/// garbage collection is always run. The percent is calculated by dividing this
+/// field value by 100, so this field must be between 0 and 100, inclusive. When
+/// specified, the value must be greater than imageGCLowThresholdPercent.
+/// Default: 85
+/// https://kubernetes.io/docs/reference/config-api/kubelet-config.v1beta1/
+
+#[derive(Debug, Clone, Eq, PartialEq, Hash)]
+pub struct ImageGCHighThresholdPercent {
+    // All settings are brought in as Strings.
+    // Important note: custom types need to have `string_impls_for!()` after
+    // their TryFrom<&str> implementation. (this requires that the inner type
+    // of a custom Kubernetes settings type is a String)
+    inner: String,
+}
+
+impl TryFrom<&str> for ImageGCHighThresholdPercent {
+    type Error = error::Error;
+
+    fn try_from(input: &str) -> Result<Self, Self::Error> {
+        // apiserver validation 1: reject empty values
+        ensure!(
+            !input.is_empty(),
+            error::InvalidImageGCHighThresholdPercentSnafu {
+                input,
+                msg: "must not be empty",
+            }
+        );
+        // apiserver validation 2: reject values that are out of bounds
+        let parsed_input: i32 = input
+            .parse::<i32>()
+            .context(error::ParseIntSnafu { input })?;
+        ensure!(
+            // Fun fact: (IMAGE_GC_THRESHOLD_MIN..=IMAGE_GC_THRESHOLD_MAX) is a range of values in Rust
+            // and you can check if a value is contained in a given range!
```

Comment:
These sorts of comments are not ideal. You can assume the reader knows Rust and is familiar with how language constructs work.

```suggestion
            // Fun fact: (IMAGE_GC_THRESHOLD_MIN..=IMAGE_GC_THRESHOLD_MAX) is a range of values in Rust
            // and you can check if a value is contained in a given range!
```

Reply:
Agreed and removed.